### ISSUE TITLE
__dict__ not showing up in tab completion

### ISSUE THREAD

**Issue by thomasballinger** [bpython/bpython#480]
~~~.py
>>> class Foo():
...     pass
...
>>>
>>> f = Foo()
>>> f.__dic
~~~

**Comment by thomasballinger** [comment]
This should probably be added to the helper functions of AttrCompleter. `dir` doesn't return it, so it needs to be checked for explicitly.

**Comment by sebastinas** [comment]
Maybe I'm just missing something, but `__dict__` shows up for me. A Python version specific bug maybe?

**Comment by thomasballinger** [comment]
It looks like this is just old-style classes.

**Comment by llllllllll** [comment]
This issue was fixed by #511
